.env_sample

```diff
@@ -17,6 +17,7 @@ export RACK_TIMEOUT_WAIT_TIMEOUT=100_000
 export RACK_TIMEOUT_SERVICE_TIMEOUT=100_000
 
 # Heroku release slug used to bust caches on deploys
+export HEROKU_RELEASE_CREATED_AT="Optional"
 export HEROKU_SLUG_COMMIT="Optional"
```

Comment:
Instead of using "Optional" for these two, can we keep the empty? This way you can test for presence and display "Not available" or something similar in case they aren't

Reply:
Sounds good, I'll change it 👍🏽